pentagon/component/kops/files/cluster.yml.jinja

```diff
@@ -0,0 +1,68 @@
+{%- if not network_mask -%}
+{%- set network_mask = 24 -%}
+{%- endif -%}
+{%- if not third_octet -%}
+{%- set third_octet = 16 -%}
+{%- endif -%}
+apiVersion: kops/v1alpha2
+kind: Cluster
+metadata:
+  name: {{ cluster_name }}
+spec:
+  {%- if additional_policies %}
+  additional_policies: |
+   {{ additional_policies|indent(4) }}
+  {%- endif %}
+  api:
+    loadBalancer:
+      type: Public
+  authorization:
+    alwaysAllow: {}
+  channel: stable
+  cloudProvider: aws
+  configBase: s3://{{ kops_state_store_bucket }}/{{ cluster_name }}
+  dnsZone: {{ cluster_dns }}
+  etcdClusters:
+  - etcdMembers:
+    {%- for az in master_availability_zones %}
+    - instanceGroup: master-{{ az }}
+      name: {{ az }}
+    {%- endfor %}
+    name: main
+  - etcdMembers:
+    {%- for az in master_availability_zones %}
+    - instanceGroup: master-{{ az }}
+      name: {{ az }}
+    {%- endfor %}
+    name: events
+  kubernetesApiAccess:
+  - 0.0.0.0/0
+  kubernetesVersion: {{ kubernetes_version }}
+  masterPublicName: api.{{ cluster_name }}
+  networkCIDR: 172.20.0.0/16
```

Comment:
I think its the only place it is used and I am leaning toward removing it as an option altogether. I'll double check.

Reply:
It looks like it *is* used in the Terraform as well, but the cidr is hard coded in the VPN stuff. 
What are your thoughts on defaulting the ip space to 172.20.0.0/16 across the board and removing it as an option?

Reply:
I think we've had at least a couple customers where we couldn't use 172.20/16 for various reasons and I would like this to remain a CLI option. While not the standard layout we would like, if we make more than 1 VPC for a client we'll typically need to change this to avoid overlap.

Reply:
Ok, noted. We'll keep it in and I'll think on reworking it a bit. I think it will make the most sense to do something like what I _didn't_ want to do and include defaults in the base Pentagon class and remove them from the component and the component template to avoid having multiple defaults in multiple places.

Reply:
@philipcristiano  lots of changes made to accommodate the request.